### PR TITLE
expose centerzoom, public getBoundsCenterZoom and Anchor.forPos

### PR DESCRIPTION
**Pull request by lpongetti**
Hi,
i'm creating a marker cluster plugin with all feature like animation and spiderfy, and this two edits can help me for don't rewrite you code.

### CODE REVIEW

File: lib/src/map/map.dart [hunk 1]
```diff
@@ -134,7 +134,7 @@ class MapState {
     if (!bounds.isValid) {
       throw Exception('Bounds are not valid.');
     }
-    var target = _getBoundsCenterZoom(bounds, options);
+    var target = getBoundsCenterZoom(bounds, options);
```
Comment (johnpryan) [inline lib/src/map/map.dart]: I think a better way would be to expose this method on `MapControllerImpl` and `MapController` instead. Their purpose is to expose the methods on `MapState`

Comment (lpongetti) [reply lib/src/map/map.dart]: Right, my bad. now its ok

**Comment by lpongetti** [comment]
Ok i added getBoundsCenterZoom to mapcontroller

**Comment by lpongetti** [comment]
You have release version 0.6.0 from dev branch... dev branch doens't support channel stable!!

**Comment by lpongetti** [comment]
Ok, i correct some things..
1) polygon with border didn't close the polygon
2) #230 marker will removed only if totally disappear from screen

i saw better what i need and i need getBoundsCenterZoom public  from mapstate public because in my plugin i have mapstate and not mapcontroller.

File: lib/src/map/map.dart (hunk 1 above)
Comment (johnpryan) [reply lib/src/map/map.dart]: I see, thanks for the clarification.

**Pull request merged**